drivers/builtin/src/oid.c

```diff
@@ -29,13 +27,8 @@
 /*
  * Macro to generate mbedtls_oid_descriptor_t
  */
-#if 1 /* OID_INFO_STRINGS */
-#define OID_DESCRIPTOR(s, name, description)  { ADD_LEN(s), name, description }
-#define NULL_OID_DESCRIPTOR                   { NULL, 0, NULL, NULL }
-#else
 #define OID_DESCRIPTOR(s, name, description)  { ADD_LEN(s) }
```

Comment:
Now the name and description are always unused. We might want to clean this up later. There's no need to do it now, and arguably having the name is a bit of documentation that we might want to keep. But please add a note saying that the name and description are present for historical reasons and no longer used.